cmd/kubeadm/app/cmd/reset.go

```diff
@@ -147,8 +146,10 @@ func (r *Reset) Run(out io.Writer, client clientset.Interface, cfg *kubeadmapi.I
 	etcdDataDir, err := getEtcdDataDir(etcdManifestPath, cfg)
 	if err == nil {
 		dirsToClean = append(dirsToClean, etcdDataDir)
-		if err := etcdphase.RemoveStackedEtcdMemberFromCluster(client, cfg); err != nil {
-			klog.Warningf("[reset] failed to remove etcd member: %v\n.Please manually remove this etcd member using etcdctl", err)
+		if cfg != nil {
```

Comment:
Add check. Because `etcdphase.RemoveStackedEtcdMemberFromCluster(client, cfg)` function will use `cfg` and not check  it.

Reply:
nice catch.